CameraOverlay/mqtt-test.py

```diff
@@ -0,0 +1,123 @@
+import sys
```

Comment:
Since this is just a copy of the DAS mqtt test script, it's redundant to copy it over here. We can just use the one in the DAS repo.